ppx/ast_attributes.ml

```diff
@@ -423,7 +423,10 @@ let is_mel_as : attr -> bool =
   warn_if_bs ~loc txt;
   txt = "mel.as" || txt = "bs.as" || txt = "as"
 
-let has_mel_as_payload (attrs : t) = Ext_list.find_first attrs is_mel_as
+let has_mel_for_js_payload (attrs : t) =
+  Ext_list.find_first attrs (fun { attr_name = { txt; loc }; _ } ->
+      warn_if_bs ~loc txt;
+      txt = "mel.for_js")
```

Comment:
I think a mel.export might make more sense here?

Reply:
we chatted offline about reusing `mel.as`.

Reply:
Make sense also